src/libraries/Directory.Build.props

```diff
@@ -51,6 +51,8 @@
   <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true'">
     <!-- don't warn on usage of BinaryFormatter from test projects -->
     <NoWarn>$(NoWarn);SYSLIB0011</NoWarn>
+    <!-- allow nullable annotated files to be incorporated into tests without warning -->
+    <Nullable Condition="'$(Nullable)' == '' and '$(Language)' == 'C#'">annotations</Nullable>
```

Comment:
Not sure how, but seems test failures related to this change

Reply:
They are.  There are now some attributes on a method that was explicitly testing that there weren't any attributes :) Ugh.